sync/src/main/kotlin/gropius/sync/TokenManager.kt

```diff
@@ -25,59 +29,77 @@ import io.ktor.serialization.kotlinx.json.*
 @ConfigurationProperties("gropius.sync")
 data class SyncConfigurationProperties(val loginServiceBase: URI, val apiSecret: String)
```

Comment:
imho this deserves its own file as it is the main configuration of this module

Reply:
70a32864136ef95c728a9a3e886c287c8cdab784